src/ChatPage.cpp

```diff
@@ -196,6 +196,11 @@ ChatPage::ChatPage(QSharedPointer<UserSettings> userSettings, QWidget *parent)
                 }
 
                 if (hasNotifications && userSettings_->hasNotifications())
+
+                        if (userSettings_->hasAlertOnNotification()) {
+                                QApplication::alert(this);
```

Comment:
This will cause the window to alert if you have an unread message in any room, even if you only received a read receipt in that room or a non notifying message instead of a new notification in that room. I don't think that is that nice. :D

Reply:
Fixed!